.github/workflows/ghpages.yml

```diff
@@ -0,0 +1,41 @@
+name: GH Pages
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+          sudo apt-get update
+          sudo apt-get install python3-sphinx
+          python3 -m pip install setuptools
+          python3 -m pip install git+https://github.com/antmicro/sphinx_antmicro_theme
+
+    - name: Build
+      run: |
+        make -C docs html
+
+    - name: Deploy
+      if: github.ref == 'refs/heads/master'
```

Comment:
And then this should be sth like:
` if: github.ref == 'refs/heads/master' && github.event_name != 'pull_request'`
to make sure it runs only on push to master

Reply:
Done